Cargo.lock

```diff
@@ -1568,7 +1579,7 @@ dependencies = [
  "async-std",
  "async-trait",
  "cfg-if 1.0.0",
- "dashmap",
+ "dashmap 4.0.2",
```

Comment:
It looks like `dashmap` `v4.0.2` is used alongside `v5.2.0`. Is that an issue? cc @mitchmindtree

Reply:
Saw that, it got pulled in when I merged in `master`

Reply:
@JoshuaBatty do you know if this would be an issue as well?

Reply:
We are using an outdated version of `dashmap` in sway-lsp. https://github.com/FuelLabs/sway/blob/d725fcbaa35dbca1723d62836367d5309243a685/sway-lsp/Cargo.toml#L12

We can update that in a separate PR